.gitignore

```diff
@@ -2,5 +2,6 @@
 /build
 /components
 /docs
+/fxa-auth-server
```

Comment:
When testing against a local auth-server, this directory is created by this [script](https://github.com/mozilla/fxa-js-client/blob/master/tests/ci/travis-auth-server-test.sh). It doesn't need to be commited.